benchmarks/src/test/scala/org/ergoplatform/db/LDBVersionedStoreRollbackBench.scala

```diff
@@ -30,16 +23,16 @@ object VLDBStoreRollbackBench
     val toRemove =
       if (i > 1) (0 to (elemsAtStep / 2)).map(_ => key(scala.util.Random.nextInt((i - 1) * elemsAtStep))) else Seq.empty
     val version = Algos.hash(key(i))
-    store.update(toInsert, toRemove)(version)
+    store.update(version, toRemove, toInsert)
   }
 
-  private val version = store.versions.last
+  private val version = store.rollbackVersions().head
 
   val et = Utils.time {
-    val result = store.rollbackTo(version)
+    val result = store.rollback(version)
```

Comment:
the previous name was better, because default meaning of `store.rollback(version)` is like removing the given version from db.

Reply:
reverted